### DISCUSSION TITLE
where is the image for 0.13.x?

### DISCUSSION THREAD

**Discussion by ElectronicBattle** [blakeblackshear/frigate#15096] [Ask A Question] [answered]
Hello

I can point at image: ghcr.io/blakeblackshear/frigate:0.14.1  but where's the equivalent for the 0.13.x releases please?

I can see https://github.com/blakeblackshear/frigate/tree/v0.13.2  but then I'm stuck.  A few searches has not landed me in the right spot.

I need to downgrade from 0.14.1 which is *brilliant* but the NAS box I am running it on is struggling.   The new way by which I can run an "app" on my TrueNAS box is to use Docker commands, hence my searching for 0.13 variants.

I keep seeing high CPU loads on the NAS box which is struggling to run 0.14 but was fine on 0.13, and the USB Coral now keeps bursting up from what used to be the usual 7 ms to sometimes a tenfold increase!

(This has all occurred due to of an upgrade in TrueNAS which I tried yesterday, which I might also need to downgrade).

Thanks!

**Comment by hawkeye217** [comment, marked as answer]
You can always find releases on the [releases page](https://github.com/blakeblackshear/frigate/releases/). Most of the time, Docker image links are published in the release notes.

For 0.13.2, they actually aren't in the release notes but you can find the main image here: `ghcr.io/blakeblackshear/frigate:0.13.2`

**Reply by ElectronicBattle** [reply]
Perfect - thanks! 

Yes, I'd looked through the releases but to no avail. Thanks for the URL for 0.13.2 which means I can fall back to a responsive solution for the time being, until I find out what my other IT problems are.

```ghcr.io/blakeblackshear/frigate:0.13.2``` noted.

**Reply by ElectronicBattle** [reply]
A minor follow-up ...my problems were due to another docker application (Motioneye) demanding too much CPU time when I'd upgraded the host OS (TreuNAS) so I ended up having to downgrade TN to the original version at which point my original Frigate (0.13.2) got reverted too. I might now try running Frigate 0.14  which is an available option.
